src/source/PeerConnection/Rtp.c

```diff
@@ -278,7 +278,15 @@ STATUS writeFrame(PRtcRtpTransceiver pRtcRtpTransceiver, PFrame pFrame)
     bufferAfterEncrypt = (pKvsRtpTransceiver->sender.payloadType == pKvsRtpTransceiver->sender.rtxPayloadType);
     for (i = 0; i < pPayloadArray->payloadSubLenSize; i++) {
         pRtpPacket = pPacketList + i;
-
+        if (pKvsRtpTransceiver->pKvsPeerConnection->twccExtId != 0) {
+            pRtpPacket->header.extension = TRUE;
+            pRtpPacket->header.extensionProfile = TWCC_EXT_PROFILE;
+            pRtpPacket->header.extensionLength = SIZEOF(UINT32);
+            UINT16 twsn = (UINT16) ATOMIC_LOAD(&pKvsRtpTransceiver->pKvsPeerConnection->transportWideSequenceNumber);
```

Comment:
is there a "declare at the top" religion? i wanna read the other commandments ;)

Reply:
Haha, actually as is It will fail for C89 compile with -pedantic flag, variable declaration is required to be at the top of a {} block (doesn't actually have to be at top of method, but that's the standard for our code.  See: https://stackoverflow.com/questions/288441/variable-declaration-placement-in-c

Reply:
Older compilers would simply fail. Also, there are small footprint compilers that do two-pass compilation for the AST+ that will fail